GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -39,47 +39,49 @@ public static string FindFileInFolders(string fileName, params string[] location
                     continue;
                 }
 
+                string fullName = string.Empty;
```

Comment:
nit: you don't need to assign the value here, as it is definitely assigned by the first `out` usage.

Reply:
You're right. I'll do you one better - it can be inlined. :)